doc/developer-notes.md

```diff
@@ -417,6 +417,21 @@ Source code organization
 
   - *Rationale*: Avoids symbol conflicts
 
+- Terminate namespaces with a comment (`// namespace mynamespace`). The comment
```

Comment:
out of scope imo

Reply:
Not really, at least not out of scope for the PR (only out of scope for the commit). In many cases we're backporting *intent* as well instead of only code, which means we have to apply the same set of changes to Dash specific code as well.